cloud/gcp/storage/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "storage_name" {
+    value = google_storage_bucket.file-storage.name
+}
+
+output "storage_id" {
+    value = google_storage_bucket.file-storage.id
```

Comment:
nit (blocking): inconsistent indentation here with the other files

Reply:
Done